### DISCUSSION TITLE
nvim stop responding to commands after pasting, but only print out raw ASCII code

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#869] [Q&A]
如复制以下代码：
```python
def taxi_fare(distance):
    if distance <= 3:
        fare = 14
    elif distance > 3 and distance <= 15:
        fare = 14 + (distance - 3) * 2.3
    else:
        fare = 14 + 12 * 2.3 + (distance - 15) * 3.5
    return fare

print(taxi_fare(10))
```

会出现以下错误，，所有按键都，命令格式，，其他模式都试下，如截图

![image](https://github.com/ayamir/nvimdots/assets/110797126/cfa8ab99-f46a-478b-ad53-1fcb90e1bcd3)

**Comment by charliie-dev** [comment]
It's not the issue of this config, it's between your system clipboard, terminal and possibly your ssh connection and remote machine. 

Can't give any advice if no further info is provided.

**Comment by charliie-dev** [comment]
Also, u really need to work on how to ask a proper question.

**Comment by Jint-lzxy** [comment]
And FYI: #489

**Reply by wxzvim** [reply]
OK，use p command
